test/gemm/gemm_bf16.cpp

```diff
@@ -32,131 +32,80 @@ namespace ck {
 namespace tensor_operation {
 namespace device {
 namespace device_gemm_instance {
+void add_device_gemm_xdl_c_shuffle_bf16_bf16_bf16_km_kn_mn_instances(std::vector<DeviceGemmPtr_>&);

```

Comment:
Please replace all "DeviceGemmPtr_" with "DeviceGemmNoOpPtr"

Reply:
Done in e514bf1337d15cbf070a7396fc6f8448a17dee47

Reply:
> Thanks for adding GEMM tests.
> 
> Please make sure all existing GEMM instances are added into these tests.
> 
> <img alt="image" width="890" src="https://user-images.githubusercontent.com/22615726/158665595-aba8535a-de68-472b-97e2-560c3c4c2f96.png">

Done
However, following device operation is not invalid.
IsSupportedArgument () will return false.
So I mark it now.
https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/develop/library/src/tensor_operation_instance/gpu/gemm/device_gemm_xdl_splitk_f16_f16_f16_mk_nk_mn_instance.cpp#L54